### PR TITLE
fix(jobs): Binary spool files corrupted by newline normalization

### LINKED ISSUE #2282: Binary spool files corrupted by the newline normalization.
https://github.com/zowe/zowe-cli/blob/b11995e95b6a38b2b2b5f2afbf41c6bd94d4cf08/packages/zosjobs/src/DownloadJobs.ts#L131-L132

The last paramter should be `!parms.binary && !parms.record`.

### PR DESCRIPTION
**Pull request by t1m0thyj**
**What It Does**

Fixes #2282 

**How to Test**

Reproduce the issue with the following steps (only on Windows):
1. Run a JCL job that outputs a spool file containing the `0A` (`\n`) character. For example:
    ```
    //CATFILE  EXEC PGM=BPXBATCH,REGION=0M
    //STDPARM  DD   *
    SH cat "//'IBMUSER.BINARY.DS'"
    /*
    //STDERR   DD   SYSOUT=*
    //STDOUT   DD   SYSOUT=*
    ```
2. Download spool in binary mode with `zowe jobs download output <jobId> --binary>`
3. Verify that `0A` (`\n` = LF) characters do not have `0D` (`\r` = CR) prepended.

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by awharn — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 

Should we get this other PR merged so that we can release a minor release to the `latest` tag with the TSO features and this fix combined?
- https://github.com/zowe/zowe-cli/pull/2272

File: packages/cli/src/zostso/issue/command/Command.handler.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
- * https://www.eclipse.org/legal/epl-v20.html
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Copyright Contributors to the Zowe Project.
- *
- */
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
```
Comment (anaxceron) [inline packages/cli/src/zostso/issue/command/Command.handler.ts:4]: Please end sentence w/ a period.

File: packages/imperative/src/imperative/src/config/cmd/secure/secure.handler.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
- * https://www.eclipse.org/legal/epl-v20.html
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Copyright Contributors to the Zowe Project.
- *
- */
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
```
Comment (anaxceron) [inline packages/imperative/src/imperative/src/config/cmd/secure/secure.handler.ts:4]: Please end sentence w/ a period. Same comment for all the other references that use the same sentence in this PR.

File: packages/zosjobs/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS jobs SDK package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed error in `DownloadJobs.downloadSpoolContentCommon` causing binary spool files to be corrupted by newline normalization. [#2282](https://github.com/zowe/zowe-cli/issues/2282)
```
Comment (anaxceron) [inline packages/zosjobs/CHANGELOG.md:7]: Can we say what DownloadJob.downblahblah is? As in: "Fixed an error in the `bbbb` function/option/cat treat that corrupted binary spool files by newline normalization."

**Review by anaxceron — COMMENTED** [review]
Left some comments requesting minor changes.

File: packages/cli/src/zostso/issue/command/Command.handler.ts (hunk 1 above)
Comment (t1m0thyj) [reply packages/cli/src/zostso/issue/command/Command.handler.ts:4]: These license headers are automatically added to the top of every file in the repo. I think we should handle any formatting changes in a separate issue.

**Review by gejohnston — APPROVED** [review]

**Review by anaxceron — APPROVED** [review]
all good @t1m0thyj, thank ya

**Review by awharn — APPROVED** [review]

**Pull request merged**